radian/settings.py

```diff
@@ -56,6 +56,7 @@ def load(self):
         self._load_setting("vi_mode_prompt", VI_MODE_PROMPT)
         self._load_setting("stderr_format", STDERR_FORMAT)
 
+        self._load_setting("add_spaces_around_equals", True, bool)
```

Comment:
Maybe better to have a setting name with prefix `completion`. Maybe something like `completion_adding_spaces_around_equals`!?

Reply:
Your suggestion sounds good. Thank you!

It turns out that RStudio has an option to disable adding spaces around equals. The same was also available for RTVS for VS2017.

![image](https://user-images.githubusercontent.com/4283816/80873463-4f27d580-8c7e-11ea-8cd0-e1c5a895f717.png)

The reason I like this form is that it makes my compact. I think that's a personal preference though.